### PR TITLE
Remove obsolete excludes for junit vintage

### PR DESCRIPTION
**Pull request by dsyer**
Spring Boot 2.7 does not have the vintage engine dependency
so it is being excluded in a bunch of places for no reason.

### CODE REVIEW

File: docs/en/server/testing.md [hunk 1]
```diff
@@ -9,14 +9,15 @@ Please refer to [Tests with Grpc-Stubs](../client/testing.md).
...
-- [gRPCurl](#grpcurl)
+- [Testing the Service](#testing-the-service)
+  - [Introductory Words](#introductory-words)
+  - [The Service to Test](#the-service-to-test)
+  - [Useful Dependencies](#useful-dependencies)
+  - [Unit Tests](#unit-tests)
+    - [Standalone Tests](#standalone-tests)
+    - [Spring-based Tests](#spring-based-tests)
+  - [Integration Tests](#integration-tests)
+  - [gRPCurl](#grpcurl)
```
Comment (ST-DDT) [inline docs/en/server/testing.md:20]: Please revert the unrelated TOC changes.

File: docs/zh-CN/client/testing.md [hunk 2]
```diff
@@ -6,11 +6,18 @@
...
+  - [要测试的组件](#要测试的组件)
+  - [有用的依赖项](#有用的依赖项)
+  - [使用 Mocked Stub](#使用-mocked-stub)
+    - [优点](#优点)
+    - [缺点](#缺点)
+    - [实现](#实现)
+  - [运行一个虚拟服务](#运行一个虚拟服务)
+    - [优点](#优点-1)
+    - [缺点](#缺点-1)
+    - [实现](#实现-1)
```
Comment (ST-DDT) [inline docs/zh-CN/client/testing.md:20]: Same here

File: docs/zh-CN/server/testing.md [hunk 3]
```diff
@@ -8,14 +8,15 @@
...
-- [集成测试](#集成测试)
-- [gRPCurl](#grpcurl)
+  - [前言](#前言)
+  - [测试服务](#测试服务-1)
+  - [有用的依赖项](#有用的依赖项)
+  - [单元测试](#单元测试)
+    - [独立测试](#独立测试)
+    - [基于Spring的测试](#基于spring的测试)
+  - [集成测试](#集成测试)
+  - [gRPCurl](#grpcurl)
```
Comment (ST-DDT) [inline docs/zh-CN/server/testing.md:19]: Same here. Maybe?

**Review by ST-DDT — APPROVED** [review]

**Review by yidongnan — COMMENTED** [review]
LGTM
